### ISSUE TITLE
Input toggles remain highlighted after being switched off

### ISSUE THREAD

**Issue by shuyuan-liu** [tomcl/issie#112]
On Issie v2.2.9 on Windows 11 build 22000, input toggles in the Simulation tab remain coloured after being turned off, until the user clicks somewhere else to remove focus from the toggle. A GIF is attached to show this effect:

![2021-10-29-09-43-39](https://user-images.githubusercontent.com/41851763/139404911-b8335449-cf4a-4e9f-89c3-9ef6f4e5efed.gif)

A minimal circuit was used, consisting of an input directly connected to an output, since this issue isn't specific to any circuit.

In the recording above the input A was initially off. Four steps were then performed:
1. Click on button to turn A on
2. Click on button again to turn A off -> button remains highlighted
3. Click on button to turn A on, then click in empty space -> button remains green but highlighting "outline" is removed
4. Click on button again to turn A off, then click in empty space -> button no longer highlighted

Seeing in the Outputs section that highlighting directly corresponds to the state of an output, it is potentially confusing that an input button can remain highlighted while being off.

**Comment by tomcl** [comment]
Thanks for this! No-one noticed it the last time we did a color change.

I have checked on W10 (the dev machine). This error also occurs. The view does not get refreshed in some cases when it should be - leading to the problem of a persistent highlight. The lack of refresh may be a complex react virtual DOM issue, in which case not quickly addressed. However a quick fix would be to make the hover color different from the '1' color.

That is clearly confusing. I will change colors next minor bugfix and hope for a complete fix but if not at least make it less confusing.
